src/views/About.vue

```diff
@@ -62,6 +62,51 @@
           The accurate estimation of water use nationally at the HUC 12 watershed scale provides a consistent national source of water use data for the first time. 
         </p>
       </div>
+      <div class="application" id="operationalPipeline">
+        <h3>Operational Pipeline</h3>
+        <p>
+          In order to update the water storage and water temperature applications for near-real time conditions, 
+          a series of steps are completed daily. The major steps in the operational pipeline include: 1) obtaining 
+          and formatting gridded climate inputs, 2) hydrologic model execution, 3) post-processing of model outputs, 
+          and 4) updating the water storage and water temperature visualizations. Climate inputs of daily maximum and 
+          minimum air temperature and precipitation accumulation from the GridMET dataset (<a href="https://doi.org/10.1002/joc.3413" target="_blank">Abatzoglou, 2013</a>) 
+          are currently used in the hydrologic model simulations that are used for the water storage and stream temperature applications. 
+          A containerized workflow has been implemented for this process to allow for modularization and flexibility of various 
+          components, as well as allowing for efficient troubleshooting. 
+        </p>
+        <div id="diagram">Diagram</div>
+        <p>
+          Each container in the workflow, as shown in the accompanying diagram, executes a specific, independent, task. 
+          The first container checks to see if the climate inputs have been updated for the latest complete day. The second 
+          container downloads the climate inputs and processes the data from their native grid to the model spatial units using 
+          a contributing area-weighted averaging algorithm. The third container executes the hydrologic model using with the initial 
+          conditions (restart file) from the previous model run and the updated climate data. The fourth container updates the initial 
+          conditions files for the next time that the model is executed. The fifth through nineth containers access the converted model 
+          outputs and update the visualizations of water storage and water temperature.
+        </p>
+        <p>
+          To complete step 4 of the operational pipeline and produce the daily-updating vector tileset to visualize modeled water storage 
```

Comment:
hmm this text seems outdated/incorrect, i'd pause on this PR until we wrap up changes to the text.